src/app/observation/observation.component.ts

```diff
@@ -19,18 +20,19 @@ export class ObservationComponent implements OnInit, isDeactivatable {
   saveQuestioner: boolean = false;
   showDetails = false;
   isOnline:any;
-  private onlineStatus: boolean = true;
+  onlineStatus: boolean = true;
 
   constructor(private navCtrl: NavController, private profileService: ProfileService,
     private utils: UtilService, private toast:ToastService,
     private alertService: AlertService) { 
+      this.toast.dismissToast();
 
       const onlineEvent = fromEvent(window, 'online').pipe(map(() => true));
       const offlineEvent = fromEvent(window, 'offline').pipe(map(() => false));
       merge(onlineEvent, offlineEvent).pipe(startWith(navigator.onLine)).subscribe((isOnline:any) => {
         this.onlineStatus = isOnline;
         if (!this.onlineStatus) {
-          this.toast.presentToast('NETWORK_OFFLINE', 'danger');
```

Comment:
why have we removed this?

Reply:
When we come form home page to observation page, 2 toasters are showing up, i have handled the toaster in observation interceptor code.

To avoid conflicts i have removed it